components/login/login.tsx

```diff
@@ -736,82 +736,85 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
                         </div>
                     )}
                 </div>
-                <div className={classNames('login-body-card', {'custom-branding': enableCustomBrand, 'with-error': hasError})}>
-                    <div
-                        className='login-body-card-content'
-                        onKeyDown={onEnterKeyDown}
-                        tabIndex={0}
-                    >
-                        <p className='login-body-card-title'>
-                            {getCardTitle()}
-                        </p>
-                        {enableCustomBrand && getMessageSubtitle()}
-                        {alertBanner && (
-                            <AlertBanner
-                                className='login-body-card-banner'
-                                mode={alertBanner.mode}
-                                title={alertBanner.title}
-                                onDismiss={alertBanner.onDismiss ?? dismissAlert}
-                            />
-                        )}
-                        {enableBaseLogin && (
-                            <div className='login-body-card-form'>
-                                <Input
-                                    ref={loginIdInput}
-                                    name='loginId'
-                                    containerClassName='login-body-card-form-input'
-                                    type='text'
-                                    inputSize={SIZE.LARGE}
-                                    value={loginId}
-                                    onChange={handleInputOnChange}
-                                    hasError={hasError}
-                                    placeholder={getInputPlaceholder()}
-                                    disabled={isWaiting}
-                                    autoFocus={true}
+                <div className='login-body-action'>
```

Comment:
When custom branding is applied the alignment between `login-body-message` and `login-body-card` is also changing:

Before:
<img width="1190" alt="Screen Shot 2022-09-07 at 10 13 32 AM" src="https://user-images.githubusercontent.com/79058848/188915781-79b7cee6-3b68-48c2-9628-2cc318b0d17e.png">

After:
<img width="1195" alt="Screen Shot 2022-09-07 at 10 15 10 AM" src="https://user-images.githubusercontent.com/79058848/188915817-55af8c12-1ca1-4a86-b1b1-b20dfd290291.png">

Maybe compensating `alternate-link`'s height in the `login-body-message` side can help keep the expected alignment, but take into account that `alternate-link` is not always present (ie. No open server enabled)

Reply:
Added `margin-top: 40 px` to `login-body-message`. 
Didn't had to add conditional css as `login-body-message` is invisible on smaller screens where `alternate-link` moves away to top right corner or is invisible.